cleanlab_studio/cli/types.py

```diff
@@ -20,6 +20,7 @@ class DatasetFileExtension(Enum):
     xls = ".xls"
     xlsx = ".xlsx"
     json = ".json"
+    pkl = ".pkl"
```

Comment:
is this used?

Reply:
ah not anymore. this class is actually used in the backend for one of the download functions, but i'm no longer using that

thanks, i'll remove